tests/kernel/mem_protect/mem_map/src/main.c

```diff
@@ -165,13 +165,28 @@ void test_z_phys_map_side_effect(void)
 	ztest_test_fail();
 }
 
+/**
+ * Show that mapping works failed as expected
```

Comment:
Please add "@brief" at begining of the line.

Reply:
@LeiW000 Got it fix, this test case conflict due to another test case merged. I will put them in another commit.